prime-router/metadata/fhir_mapping/hl7/datatype/ContactPoint.yml

```diff
@@ -1,17 +1,32 @@
 system_1:
-     condition: $emailValue NULL
-     type: STRING
-     valueOf: "phone"
-     vars:
-         emailValue: XTN.4
-
-system_2:
      condition: $emailValue NOT_NULL
      type: STRING
      valueOf: "email"
      vars:
          emailValue: XTN.4
 
+system_2:
+  condition: $useCode EQUALS BPN || $equipType EQUALS BP
+  type: STRING
+  valueOf: "pager"
+  vars:
+    useCode: String, XTN.2
+    equipType: String, XTN.3
+
+system_3:
+  condition: $equipType EQUALS_STRING FX
+  type: STRING
+  valueOf: "fax"
+  vars:
+    equipType: String, XTN.3
+
+system_4:
+  condition: $emailValue NULL
+  type: STRING
+  valueOf: "phone"
+  vars:
+    emailValue: XTN.4
+
```

Comment:
Maybe we can use valueSets here!

Reply:
This is on the Linux4Health library, so no value sets here

Reply:
😭